pkg/spdk/controllerserver.go

```diff
@@ -539,3 +553,39 @@ func newControllerServer(d *csicommon.CSIDriver) (*controllerServer, error) {
 
 	// return &server, nil
 }
+
+func GetCryptoKeys(ctx context.Context, pvcName, pvcNamespace string) (cryptoKey1, cryptoKey2 string, err error) {
```

Comment:
we might need to add a clusterRole to get secrets from the namespace.

Reply:
A clusterrole already exist [here](https://github.com/simplyblock-io/spdk-csi/blob/master/charts/spdk-csi/templates/controller-rbac.yaml#L24)